repos/system_upgrade/common/actors/checkdnfpluginpath/libraries/checkdnfpluginpath.py

```diff
@@ -0,0 +1,28 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import DnfPluginPathDetected
+
+DNF_CONFIG_PATH = '/etc/dnf/dnf.conf'
+
+
+def check_dnf_pluginpath(dnf_pluginpath_detected):
+    """Create an inhibitor when pluginpath is detected in DNF configuration."""
+    if dnf_pluginpath_detected.is_pluginpath_detected:
+        reporting.create_report([
+            reporting.Title('DNF configuration uses pluginpath option.'),
+            reporting.Summary(
+                'The option "pluginpath" is set in {}. Localisation of dnf plugins is not constant between '
+                'system releases which can cause issues with the upgrade. Remove the option or comment it out '
+                'before running the upgrade.'
+                .format(DNF_CONFIG_PATH)
```

Comment:
This needs to be updated as the text is not clear. The set value is constant - which is the problem, because the path where the plugins are stored is changed during the upgrade (different python versions). And we expect this path to be pointing to RHEL plugins really.